packages/app-admin-ui/client/pages/Item/Footer.js

```diff
@@ -75,38 +77,40 @@ function Reset({ canReset, onReset }) {
   );
 }
 
-export default memo(function Footer(props) {
-  const { onSave, onDelete, canReset, updateInProgress, onReset, hasWarnings, hasErrors } = props;
-  const cypressId = 'item-page-save-button';
+export default memo(
```

Comment:
I believe all the changes in this file can now be safely reverted 👍

Reply:
this uses `list.access.update/delete` to disable update and delete button.

Reply:
Oh, sorry, my bad, I should have turned whitespace off and read the diff properly!

Reply:
alternate way is to pass `canSave` and `canDelete` to Footer.